test/alphabet/gap/gapped_alphabet_test.cpp

```diff
@@ -43,74 +43,173 @@
 
 using namespace seqan3;
 
```

Comment:
For the test cases, please try to maintain order between dependent test cases, i.e. if tests have to depend on each other, write them in a way where that only depend on previously tested functionality, not on functionality tested later.
Thus in case of an error, the first failed test will be the one responsible and one doesn't have to sort through all of them.
I am also fine with you just testing special behaviour of the class here, since we will add generic tests for constructors and to_char et cetera soon.